src/com/oltpbenchmark/benchmarks/tpcc/procedures/NewOrder.java

```diff
@@ -266,19 +255,13 @@ private void newOrderTransaction(int w_id, int d_id, int c_id,
       if (!rs.next())
         throw new RuntimeException("C_D_ID=" + d_id
             + " C_ID=" + c_id + " not found!");
-      c_discount = rs.getFloat("C_DISCOUNT");
```

Comment:
The tpcc spec says that the values have to be stored somewhere.
I guess thats why they did it this way.

Retain all the unused varaibles as well. Let the compiler optimize it out if needed.

Reply:
done, in all procedures classes